src/SingleInvoker.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import { Auth } from "src/Auth.sol";
+
+/// @title SingleInvoker
+/// @author Julian Rachman <https://github.com/Jrachman>
+/// @notice Example EIP-3074 Invoker contract which executes a single call using AUTH and AUTHCALL.
+contract SingleInvoker is Auth {
```

Comment:
per convo in DMs,
- rebase off #11 and employ the same tactics in your tests to get a better comparison of gas metering between the two
- potentially rebase off #12 to remove shared Invoker boilerplate